test/unit/nft-api.test.ts

```diff
@@ -1204,6 +1209,54 @@ describe('NFT module', () => {
     });
   });
 
+  describe('computeRarity()', () => {
+    const contractAddress = '0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d';
+    const tokenId = '7495';
```

Comment:
one more request: can you use a numerical value for `tokenId` here to check that the logic to convert `Bignumberish` input to a string is properly executed? Thanks!

Reply:
Yup!

Reply:
Ready 👌🏼